lms/envs/aws.py

```diff
@@ -686,3 +686,6 @@
 if FEATURES.get('ENABLE_LTI_PROVIDER'):
     INSTALLED_APPS += ('lti_provider',)
     AUTHENTICATION_BACKENDS += ('lti_provider.users.LtiBackend', )
+
+############ BATCH-GENERATED CERTIFICATES FROM INSTRUCTOR DASHBOARD ###########
+CERTIFICATES_INSTRUCTOR_GENERATION = ENV_TOKENS.get('CERTIFICATES_INSTRUCTOR_GENERATION')
```

Comment:
I don't think this is right, 'CERTIFICATES_INSTRUCTOR_GENERATION' is a FEATURE flag and - as such - is part of the subdict that is already read from ENV_TOKENS in aws.py.

You should remove this line.